# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
# Commit: b94935661622
# Full Hash: b949356616223660996acf39e1e353e7bbd2a60d
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-03 09:44:53
# Regressor Bug: 1376873
# File Overlap Count: 4
# Description:
#   Bug 1376873 - Use Call interface in AudioConduit; r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D7441
# ==============================================================================

diff -r 6eca45c83131 -r b94935661622 media/webrtc/signaling/gtest/mediapipeline_unittest.cpp
--- a/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Mon Mar 26 16:19:23 2018 -0400
+++ b/media/webrtc/signaling/gtest/mediapipeline_unittest.cpp	Tue Aug 21 13:39:53 2018 -0400
@@ -236,7 +236,9 @@
  public:
   TestAgent() :
       audio_config_(109, "opus", 48000, 960, 2, 64000, false),
-      audio_conduit_(mozilla::AudioSessionConduit::Create()),
+      audio_conduit_(mozilla::AudioSessionConduit::Create(
+        WebRtcCallWrapper::Create(),
+        test_utils->sts_target())),
       audio_pipeline_(),
       transport_(new LoopbackTransport) {
   }